readme.md

```diff
@@ -104,7 +106,23 @@ systemd/[supervisord](http://supervisord.org/) or put it into a docker.
 
 ### Run in Docker
 
-// TBD
+```bash
+mkdir lobbyboy_data
+# Generate a config file
+docker run --rm ghcr.io/laixintao/lobbyboy lobbyboy-config-example > lobbyboy_data/config.toml
+# Generate an ssh key pair
+mkdir lobbyboy_data/.ssh
+ssh-keygen -f lobbyboy_data/.ssh/id_rsa
+...
+# Run the docker container
+docker run -v `pwd`/lobbyboy_data/:/app/dev_datadir -p "12200:12200" -d ghcr.io/laixintao/lobbyboy
+```
+
+The lobbyboy server should be active on 12200 port and you can connect to it with
+
+```
+ssh Gustave@127.0.0.1 -p 12200 -i lobbyboy_data/.ssh/id_rsa
+```
```

Comment:
I don't think users can run successfully follow this.

there are 2 pairs:
- lobbyboy server authenticate ssh user: using authorized_keys in config: https://github.com/laixintao/lobbyboy/blob/ecab1eea2c744834665fdfe2f6318e3d2f92b627/lobbyboy/conf/lobbyboy_config.toml#L18 , so the key pair you generated by `ssh-keygen -f lobbyboy_data/.ssh/id_rsa` won't be used.
- ssh user verify lobbyboy server: lobbyboy's key will be generated when server starts, only if not exist. here: https://github.com/laixintao/lobbyboy/blob/ecab1eea2c744834665fdfe2f6318e3d2f92b627/lobbyboy/socket_handle.py#L127

So for saving users' trouble, I suggest that we commit the ssh keys in git, so users can just start a docker, then ssh into it (always).

```
mkdir lobbyboy_data/.ssh
ssh-keygen -f lobbyboy_data/.ssh/id_rsa
````

those won't be needed.

---

I think we need to print a warning message when the server start with the default trusted pub key, and refused to listen on 0.0.0.0 for safety consideration.

Reply:
> there are 2 pairs:

I don't see why there are TWO pairs, from my testing I must add the pubkey into `authorized_key` and use `ssh -i private_key` to connect. The existing one in the example config seems not used.

Reply:
hmm sorry, I just checked the code. seems @luxiaba 's refactor changed this behavior.

in short:

the `data_dir/.ssh/id_rsa` is not for you to use this key to login, instead, it is the server's key, which lobbyboy will show you (the pubkey finger print actually), for you to trust it (there is no middle man).

Reply:
this is too confusing, let me check it back.

Reply:
I have submitted a PR to you:

https://github.com/frostming/lobbyboy/pull/1